reference_data/centos7.yml

```diff
@@ -0,0 +1,116 @@
+---
+- name: CentOS7
+  distr_type: rhel
+  distr_version: '7'
+  #? is this email correct ?#
+  contact_mail: packager@centos.org 
+  copyright: Copyright {{ year }} CentOS
+  test_dist_name: centos7
+  modularity:
```

Comment:
Centos 7 does not have modules

Reply:
@maccelf shold I remove all block modulatiry alltoghether?

Reply:
I think yes